services/provider/server/server.go

```diff
@@ -1229,13 +1230,39 @@ func (s *OCSProviderServer) getDesiredCephConnection(
 		return nil, fmt.Errorf("failed to extract monitor IPs from configmap %s: %v", monConfigMap, err)
 	}
 
+	var readAffinity *csiopv1a1.ReadAffinitySpec = nil
+	readAffinityOptions := util.GetReadAffinityOptions(storageCluster)
+
+	readaffinityDisabledAnnotation := consumer.Annotations["ocs.openshift.io/disable-read-affinity"] == "true"
+	defaultCrushLocationLabels := []string{
+		"topology.rook.io/chassis",
+		"topology.rook.io/rack",
+		"topology.rook.io/row",
+		"topology.rook.io/pdu",
+		"topology.rook.io/pod",
+		"topology.rook.io/room",
+		"topology.rook.io/datacenter",
+		"topology.kubernetes.io/region",
+		"topology.kubernetes.io/zone",
+		"kubernetes.io/hostname",
+	}
```

Comment:
There is no need to define the slice out side of the if below, please move inside the if statement

Reply:
done